status-im/status-react
# fix: routes ui for bridge flow

fixes #19970

### Summary

This PR fixes broken UI for routes on bridge flow (+ button should not be displayed in `To` column, also selected network is not showing). Also added new unit tests on relevant utils functions to cover these cases.

### Testing notes

Bridge flow seems broken and user can't confirm Bridge operation, this PR only fixes routes ui.

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Login
- Go to wallet
- Go to account
- Tap on Bridge
- Select a network to bridge to
- Enter a valid amount
- Check routes UI are displayed correctly

### Before and after screenshots comparison

#### Before

https://github.com/status-im/status-mobile/assets/18485527/9004400b-2faf-4d7b-afe6-4ac7fd277bf8

#### After

https://github.com/status-im/status-mobile/assets/18485527/2691e363-f1b0-46b4-80dc-b1c361d2a8d6

status: ready
